src/components/app.tsx

```diff
@@ -189,39 +216,62 @@ export const App: React.FC<IAppProps<IModelInputState, IModelOutputState, IModel
   const backgroundImage = currentEnvironment?.backgroundImage;
   const leafPackState = LeafPackStates.find((ls) => ls.leafDecomposition === leafDecomposition) || LeafPackStates[0];
 
-  const [trayAnimals, setTrayAnimals] = useState<TrayAnimal[]>([]);
+  const [trayObjects, setTrayObjects] = useState<TrayObject[]>([]);
   const [showTray, setShowTray] = useState(false);
+
   const handleRewind = () => {
     setShowTray(false);
-    setTrayAnimals([]);
+    setTrayObjects([]);
+    setTraySelectionType(undefined);
     rewindSimulation();
   };
 
   const [traySelectionType, setTraySelectionType] = useState<TrayType | undefined>(undefined);
   const handleTrayObjectSelect = (objectType: TrayType) => {
-    if (objectType !== LeafType.leaf) {
+    if (objectType !== LeafType.birch && objectType !== LeafType.oak && objectType !== LeafType.maple) {
```

Comment:
Seems like a good place for an `isLeaf()` utility function that can be extended if/when new leaves are added to the collection.

Reply:
That's going to come in the next PR.  The leaves are all temporary and will be replaced by a set of 21 new leaves and leaf types.  But I will definitely add a helper when I work on leaves in the next PR.